crates/libs/core/src/types/runtime/health.rs

```diff
@@ -0,0 +1,72 @@
+use mssf_com::FabricTypes::{FABRIC_HEALTH_INFORMATION, FABRIC_HEALTH_REPORT_SEND_OPTIONS};
+use windows_core::PCWSTR;
+
+use crate::{strings::HSTRINGWrap, types::HealthState, HSTRING};
+
+pub type SequenceNumber = i64;
+
+/// FABRIC_HEALTH_INFORMATION
+#[derive(Debug, Clone)]
+pub struct HealthInformation {
+    pub source_id: HSTRING,
+    pub property: HSTRING,
```

Comment:
Semantically, what sort of information should be placed in the source_id and property fields? The HSTRING type itself and even the names are not super clear what the expectations are for these values.

Reply:
added documentation